README.md

```diff
@@ -1,94 +1,71 @@
-# ![Async Logo](https://raw.githubusercontent.com/asyncjs/assets/master/branding/logo/async-logo-32x32.png) AsyncJS Assets
+![Async Logo](./branding/logo/logo.png)
 
-**A collection of AsyncJS assets**
+This repository contains Async brand assets and guidelines. Feel free to use and customize these assets for your articles, talks, websites, and anything else you can think of.
 
-## Branding
+<p align="center">
+  <a href="#logo">Logo</a>&nbsp;&nbsp;&nbsp;
+  <a href="#colours">Colours</a>&nbsp;&nbsp;&nbsp;
+  <a href="#fonts">Fonts</a>&nbsp;&nbsp;&nbsp;
+</p>
 
-### Colors
+## Logo
 
-| Name       | Hex Color Code | Preview |
-|:-----------|---------------:|:-------:|
-| Black      | `#000000`      | ![swatch](https://raw.githubusercontent.com/asyncjs/assets/master/color-swatches/swatch-black.png) |
-| Red        | `#E52222`      | ![swatch](https://raw.githubusercontent.com/asyncjs/assets/master/color-swatches/swatch-red.png) |
-| Grey       | `#9D999E`      | ![swatch](https://raw.githubusercontent.com/asyncjs/assets/master/color-swatches/swatch-grey.png) |
-| Grey light | `#AAAAAA`      | ![swatch](https://raw.githubusercontent.com/asyncjs/assets/master/color-swatches/swatch-grey-light.png) |
+The Async logo was designed and built by [Jake 'Sid' Smith](https://twitter.com/jakesidsmith) (<https://jakesidsmith.com/>). This work and all content on our website are licensed under [Creative Commons Attribution 3.0 Unported (CC BY 3.0)
```

Comment:
Just because this is the attribution I use everywhere (in case I lose my domain).

```suggestion
The Async logo was designed and built by [Jake 'Sid' Smith](https://twitter.com/jakesidsmith) (<https://github.com/jakesidsmith>). This work and all content on our website are licensed under [Creative Commons Attribution 3.0 Unported (CC BY 3.0)
```